source/assets/js/playground.ts

```diff
@@ -0,0 +1,229 @@
+import { setDiagnostics } from '@codemirror/lint';
+import { Text } from '@codemirror/state';
+import { EditorView } from 'codemirror';
+import debounce from 'lodash/debounce';
```

Comment:
@jamesnw I replaced `lodash.debounce` with this, since it should result in the same bundle size and Lodash says the per-method packages are deprecated: https://lodash.com/per-method-packages